### PR TITLE
List styling

### PR DESCRIPTION
**Pull request by gprocell927**
**What is not working: ** Saving a grocery list does not export the list as a PDF. Sadness. 😭 
**What _is_ working:  ** Style/theme from main app implemented on list. When new browserWindows open, windows are tiled so that the user knows there are 2 windows available.

### CODE REVIEW

File: grocery-list.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
       "quantity": "one dozen"
     },
     {
-      "item": "Peanuts",
-      "quantity": "1 ton"
+      "item": "cookies",
+      "quantity": "all of them"
```
Comment (stephanieea) [inline grocery-list.json:13]: ALL THE COOKIES!!! 🍪

File: lib/main.js [hunk 2]
```diff
@@ -26,18 +26,15 @@ menubar.on('after-create-window', () => {
...
-  // })
-//  menubar.window.webContents.openDevTools();
+  )
 })
 
 const createGroceryList = exports.createGroceryList = (file) => {
-  let groceryWindow = new BrowserWindow({show: false})
+  let groceryWindow = new BrowserWindow({
+    show: false,
+    x:0,
```
Comment (stephanieea) [inline lib/main.js:35]: What're the x and y coordinates doing?

File: lib/main.js [hunk 3]
```diff
@@ -55,8 +52,20 @@ const createGroceryList = exports.createGroceryList = (file) => {
   return groceryWindow
 }
 
+const saveFile = (content) => {
```
Comment (stephanieea) [inline lib/main.js:55]: I like it!!!

**Review by stephanieea — APPROVED** [review]

File: grocery-list.json (hunk 1 above)
Comment (apsitos) [reply grocery-list.json:13]: But if we don't get 1 ton of peanuts, what will we feed the elephants? :peanuts:

**Review by apsitos — APPROVED** [review]

**Comment by apsitos** [comment]
:woman_cook:

**Pull request merged**